src/QsCompiler/CompilationManager/EditorSupport/SymbolInformation.cs

```diff
@@ -244,7 +243,7 @@ bool OverlapsWithPosition(Range symRange)
                     return false;
                 }
                 referenceLocations = parent.Specializations
-                    .Where(spec => spec.SourceFile == file.FileName)
+                    .Where(spec => spec.Source.AssemblyOrCode == file.FileName)
```

Comment:
Is `spec.Source.AssemblyOrCode == file.FileName` correct? Should this be `spec.Source.CodeFile == file.FileName` instead?

Reply:
Looking at the code a bit more it seems like `spec.Source.AssemblyOrCode == file.FileName` is correct. I'll just wait for confirmation in case I have misread the code again :)

Reply:
Yes, `Source.AssemblyOrCode` was made to be backwards compatible with the original `SourceFile` property. It should always be OK to replace usages of `SourceFile` with it. I think if we didn't use it here, it would be something like `spec.Source.AssemblyFile.IsNull && spec.Source.CodeFile == file.FileName`. Code file paths in references could match a local file path by coincidence.